### ISSUE TITLE
Buttons Missing (Ugly Close Button)

### ISSUE THREAD

**Issue by Maiq-The-Liar** [rudrab/Shadow#145]
First of all, i Love this Theme!! Keep it up.
![Screenshot from 2020-09-16 23-33-56](https://user-images.githubusercontent.com/71410967/93394867-2c850a80-f875-11ea-9b4d-3f5876a8b36e.png)
This close button is ugly as hell and not the pretty one you have in your Theme.
Several Buttons are missing such as the close button.
How do i get the rest of the buttons especially the close button up and running?

**Comment by rudrab** [comment]
Hi,
Thanks for reporting. This is odd. 
If you have installed by yourself (either from gnome-look or from github), can you please send me
`ls -l ~/.icons/shadow/scalable/actions | wc -l
`
and
 
`ls ~/.icons/shadow/scalable/actions/window-close-symbolic.svg
`

**Comment by Maiq-The-Liar** [comment]
Why is it green?
The first line with wc at the end i couldnt find but the terminal said 287 when i pasted it.

________________________________
Von: rudrab <notifications@github.com>
Gesendet: Donnerstag, 17. September 2020 15:32
An: rudrab/Shadow <Shadow@noreply.github.com>
Cc: Maiq-The-Liar <tobias.odermatt@hotmail.ch>; Author <author@noreply.github.com>
Betreff: Re: [rudrab/Shadow] Buttons Missing (Ugly Close Button) (#145)


Hi,
Thanks for reporting. This is odd.
If you have installed by yourself (either from gnome-look or from github), can you please send me
ls -l ~/.icons/shadow/scalable/actions | wc -l
and

ls ~/.icons/shadow/scalable/actions/window-close-symbolic.svg

**Comment by rudrab** [comment]
That's probably the problem. There are 306 icons in `actions`. You are somehow probably missing a few of them. 
Can you try reinstalling? (I can't check gnome-look right now, as the site is not openning, but I have checked `github`, and the close icon is there).

**Comment by rudrab** [comment]
> Why is it green?

This is green for other de. With gnome, you use the symbolic version, in this case, `windos-close-symbolic.svg`, not `window-close.svg`

**Comment by Maiq-The-Liar** [comment]
That is not true in my case. I downloaded it from github (again) and in scalable/ actions, there are only 286 items...
The packet seems to be incompleate. can you send me a downloadable version that is compleate?
The packet on OpenDesktop is incompleate as well.

**Comment by rudrab** [comment]
Actually I probably got your problem. Are you getting close button green with some Qt based(or KDE) app?

**Comment by Maiq-The-Liar** [comment]
Sorry but im not very familiar with Linux... so i maybe am not really understanding your question...
I downloaded the shadow-8.2.tar.xz (4.66MB) Default branch for GTK based distro like gnome. With that im getting the close button you can see in the Screenshot on all windows.
I also tried the installing the shadow-kde-04-2018.tar.xz (77.10 MB) Special branch for Qt based de, e.g. KDE. With that 
im getting an odd green (and maybe red?) close button (also not very pleasant looking).
I am currently using Ubuntu 20.04.

I hope my answer was sufficient.

Thanks for your Help!

**Comment by Maiq-The-Liar** [comment]
I think this might be interesting as well. Brighter and Darker Symbols also look weird.
Or the top right bar does as well look odd.

![Screenshot from 2020-09-29 23-11-46](https://user-images.githubusercontent.com/71410967/94616957-71636500-02a9-11eb-8663-c7001cebba41.png)
![Screenshot from 2020-09-29 23-10-08](https://user-images.githubusercontent.com/71410967/94617006-745e5580-02a9-11eb-8454-c9665a8b579a.png)
![Screenshot from 2020-09-29 23-12-11](https://user-images.githubusercontent.com/71410967/94617041-77594600-02a9-11eb-83ed-6d0169791ae1.png)

**Comment by rudrab** [comment]
The 2nd one is volume button, not brightness. What about rightbar? I feel those are as intended.

**Comment by Maiq-The-Liar** [comment]
I still feel like i'm missing a lot of icons (close button proves it).
But i downloaded so many packs and none of them worked for me since they were incomplete.
Can you send me a complete package? (via mail)
I really love this package :3

**Comment by rudrab** [comment]
hi, yes...the close button shows red border in your screen shot, which means, in ubuntu, they are probably overriding the default nameing scheme. I am yet to get the solution. I will definitely update the icon if I get the solution. On the other hand, other icons look as intended.
